2024/src/components/Footer.tsx

```diff
@@ -49,12 +50,22 @@ const LinksBox = styled.ul`
 `
 const Link = styled(_Link)`
   line-height: 1.8em;
-  color: ${({ theme }) => theme.colors.text};
+  line-height: 1.8em;
+  color: #575757;
```

Comment:
We should be using theme colors. Can you add a new one?

Reply:
Good point, just pushed this update.